engine/option.go

```diff
@@ -74,3 +78,11 @@ func WithVSNoNewMH(ok bool) Option {
 		return nil
 	}
 }
+
+// WithHttpTimeout sets http timeout for queries to DHStore
+func WithHttpTimeout(timeout time.Duration) Option {
```

Comment:
Rename to `WithHttpClientTimeout` to clarify that this is timeout for http _client_?